res/values-in/arrays.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string-array name="home_button_options">        <item>Berpindah antara kategori baku dan semua aplikasi</item>
+        <item>Berpindah antara kategori baku dan riwayat</item>
+        <item>Tidak berpindah</item>
+    </string-array>
+    <string-array name="home_button_values" translatable="false">
```

Comment:
There is no need to include strings and string arrays with translatable flag set to false, because they won't be used by program. So it can be deleted.